packages/superset-ui-number-format/src/NumberFormatterRegistry.ts

```diff
@@ -23,7 +23,11 @@ export default class NumberFormatterRegistry extends RegistryWithDefaultKey<
   }
 
   get(formatterId?: string) {
-    const targetFormat = `${formatterId ?? this.defaultKey}`.trim();
+    const targetFormat = `${
+      formatterId === null || typeof formatterId === 'undefined' || formatterId === ''
+        ? this.defaultKey
+        : formatterId
+    }`.trim();
```

Comment:
I appreciate the explicitness, but could this be simplified as `formatterId ?? (formatterId || this.defaultKey)`.

Reply:
eslint will try to change `||` to `??`